src/index.html

```diff
@@ -1,12 +1,434 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+
+    <header class="header">
+
+      <!-- Logo -->
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <!-- Navigation List -->
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#sumsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers"
+               class="nav__link"
+               data-qa="nav-hover"
+              >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+
+
+    <main class="main">
+      <div class="main__container">
+        <div class="product-card" data-qa="card">
+          <img class="product-card__image"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+
+          <div class="product-card__name">
+            <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          </div>
+
+          <div class="product-card__code">
+            <p>Product code: 195434</p>
+          </div>
+
+
+          <div class="product-card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="product-card__reviews">
+              <p>Reviews: 5</p>
+            </div>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-name">Price:</p>
+            <p class="product-card__price-number">$2,199</p>
+          </div>
+
+          <div class="product-card__button">
+            <a
+             class="product-card__button-link"
+             href="/"
+             data-qa="card-hover"
+             >
+              buy
+            </a>
+          </div>
```

Comment:
```suggestion
          <div class="product-card__button">
            <a
             class="product-card__button-link"
             href="/"
             data-qa="card-hover"
            >
              buy
            </a>
          </div>
```
Why you nedd this container at all?